src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/Constants/CertificateStatus.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace SFA.DAS.Assessor.Functions.ExternalApis.Assessor.Constants
+﻿using System.Linq;
+
```

Comment:
Apologies for this..... now that you have copied this across I can see how wrong that is - you were correct originally the definition of 'Delivered' or 'NotDelivered' is a string which belongs in the print command it should not be tied to the  CertificateStatus - which is really a co-incidence that the strings are the same.

After putting it back - could you Delete the CertificateStatus from the function app - as it really should not be there